### ISSUE TITLE
Q&A tool

### ISSUE THREAD

**Issue by michaelmarshall** [etenlab/crowd.rocks#86]
depends on #66 

since we've already done Q&A on the old site I'm not going to re-explain it here or create new wireframes right now. please bring over the Q&A functionality as best you understand it for a first draft. let me know when you finalize the database schema and I will review just the schema file at that time. Q&A will be a new tool on the home screen.

it is important for this functionality to be built on top of the document viewer code. This feature will be different than the document viewer (new page), but we want the React component that the document viewer uses to be the same component that the Q&A uses. Other features/pages will use this document viewer component as well, so it needs to be well architected.

**Comment by michaelmarshall** [comment]
the document page, Q&A page, and Pericope page need to all use the same document viewer component. the doc view page won't show any special features on top of words, but the Q&A and Pericope pages will show special stuff on the words. The Q&A page will have question and answer dots above the words, the pericope will have similar things.

So be sure to architect these components so that all the pages can use them.
